src/style.css

```diff
@@ -1,3 +1,87 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
+  font-size: 12px;
+  margin: 0;
+}
+
+header {
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
   margin: 0;
+  box-sizing: border-box;
+  font-size: 12px;
+  background-color: #fff;
+}
+
+header > a {
+  height: 40px;
+  width: 40px;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+}
+
+.nav__item {
+  margin-right: 20px;
+  position: relative;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: #000;
+  font-weight: 500;
+  text-decoration: none;
+  transition: color 0.3s;
+  position: relative;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
```

Comment:
Move all repeatable colors to CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)